server/server.go

```diff
@@ -195,6 +210,12 @@ func NewMCPServer(
 		version:              version,
 		notificationHandlers: make(map[string]NotificationHandlerFunc),
 		notifications:        make(chan ServerNotification, 100),
+		capabilities: serverCapabilities{
```

Comment:
I think the default for this should be empty. We shouldn't force anyone to implement these capabilities by default.